demos/browser/app/meeting/meeting.ts

```diff
@@ -25,8 +25,9 @@ import {
   ScreenMessageDetail,
   ScreenShareFacadeObserver,
   TimeoutScheduler,
+  Versioning,
   VideoTileState,
-} from '../../../../src/index';
+} from 'amazon-chime-sdk-js';
```

Comment:
Why is this changing? I think the original idea was that for local development this should point to the index file directly in the source folder. However, the demo package.json does use a `file:` reference, so maybe that is equivalent?

Reply:
Nope it should not change. I tested with the npm version and forgot to revert it.